### DISCUSSION TITLE
Can OpenThread Set up Network Without Any NCP Devices?

### DISCUSSION THREAD

**Discussion by AllsVoid** [openthread/openthread#9211] [Q&A] [answered]
Hi~
I follow the guide and link device through openthread successfully.

- OpenWRT with nRF52840 Dongle (ot-rcp.hex)
- Ubuntu with nRF52840 DK (ot-cli-ftd.hex)

However, I'd like to know if it is possible to establish an openthread network **without** using an ncp device (in my example, nRF52840 Dongle), and successfully establish a connection through the thread network?

Hope someone can answer it for me, thanks~

**Comment by jwhui** [comment, marked as answer]
The Thread Specification currently requires an IEEE 802.15.4 radio to directly participate in the Thread mesh network.

**Reply by AllsVoid** [reply]
Got it, I would also like to ask, is IEEE 802.15.4 established by an NCP device to establish a connection with IoT (nRF52840 dk)? 
Can I see the relevant content through the packet capture software (like Wireshark)?

**Reply by jwhui** [reply]
IEEE 802.15.4 is the link-layer wireless protocol used to communicate between devices.

There are a number of solutions to capture IEEE 802.15.4 frames and view them in Wireshark. Some examples:
- https://www.nordicsemi.com/Products/Development-tools/nrf-sniffer-for-802154
- https://openthread.io/guides/pyspinel/sniffer-extcap

**Reply by AllsVoid** [reply]
OK. Thanks~
